shared-signals/template.yaml

```diff
@@ -150,6 +234,28 @@ Resources:
       Name: CognitoTestUsersArn
       Type: String
       Value: !Ref CognitoTestUsers
+
+  IntegrationTestProducerQueueArnParameter:
```

Comment:
These will need the Condition on them as well I believe

Reply:
Just testing... Done